stripped/notebooks/Klasyfikacja-pustynia_stripped.py

```diff
@@ -0,0 +1,377 @@
+GBD = NASA_lato[(NASA_lato['lon'] >= -116) & (NASA_lato['lon'] <= -114) & (NASA_lato['lat'] >= 40) & (NASA_lato['lat'] <= 41.5)]
+
+
+"""
+##### Obszary pustynno - niepustynne
+"""
+
+CD_i_niepustynia = NASA_lato[(NASA_lato['lon'] >= -106.5) & (NASA_lato['lon'] <= -104.5) & (NASA_lato['lat'] >= 32.5) & (NASA_lato['lat'] <= 33.5)]
+CP_i_niepustynia = NASA_lato[(NASA_lato['lon'] >= -109) & (NASA_lato['lon'] <= -107) & (NASA_lato['lat'] >= 37.5) & (NASA_lato['lat'] <= 39)]
+GBD_i_niepustynia = NASA_lato[(NASA_lato['lon'] >= -115) & (NASA_lato['lon'] <= -113) & (NASA_lato['lat'] >= 42.5) & (NASA_lato['lat'] <= 44)]
+
+
+"""
+##### Obszary niepustynne
+"""
+
+niepustynia_przy_CD = NASA_lato[(NASA_lato['lon'] >= -109.5) & (NASA_lato['lon'] <= -107.5) & (NASA_lato['lat'] >= 33) & (NASA_lato['lat'] <= 34)]
+niepustynia_przy_CP = NASA_lato[(NASA_lato['lon'] >= -107) & (NASA_lato['lon'] <= -105) & (NASA_lato['lat'] >= 39) & (NASA_lato['lat'] <= 40.5)]
+niepustynia_przy_GBD = NASA_lato[(NASA_lato['lon'] >= -124) & (NASA_lato['lon'] <= -122) & (NASA_lato['lat'] >= 39.5) & (NASA_lato['lat'] <= 41)]
+
+
+"""
+# Wizualizacje + wstępne ustalenia parametrów
+"""
+
+"""
+##### GVEG (wskaźnik roślinności)
+"""
+
+fig, axes = plt.subplots(3, 3, figsize=(15, 15))
+
+def plot_histogram(ax, data, title, color):
+    ax.hist(data['GVEG'], bins=20, color=color, edgecolor="black")
+    ax.set_title(title)
+    ax.set_xlabel("GVEG")
+    ax.set_ylabel("Frequency")
+    ax.set_xlim(0, 1)
+    ax.set_ylim(0, 40)
+
+plot_histogram(axes[0, 0], CD, "CD_pustynia", "orange")
+plot_histogram(axes[0, 1], CD_i_niepustynia, "CD_pustynia/nie-pustynia", "yellow")
+plot_histogram(axes[0, 2], niepustynia_przy_CD, "CD_nie-pustynia", "green")
+
+plot_histogram(axes[1, 0], CP, "CP_pustynia", "orange")
+plot_histogram(axes[1, 1], CP_i_niepustynia, "CP_pustynia/nie-pustynia", "yellow")
+plot_histogram(axes[1, 2], niepustynia_przy_CP, "CP_nie-pustynia", "green")
+
+plot_histogram(axes[2, 0], GBD, "GBD_pustynia", "orange")
+plot_histogram(axes[2, 1], GBD_i_niepustynia, "GBD_pustynia/nie-pustynia", "yellow")
+plot_histogram(axes[2, 2], niepustynia_przy_GBD, "GBD_nie-pustynia", "green")
+
+plt.tight_layout()
+plt.show()
+
+
+GVEG_graniczne = round(
+    pd.Series([
+        CD['GVEG'].quantile(0.75),
+        niepustynia_przy_CD['GVEG'].quantile(0.25),
+        CP['GVEG'].quantile(0.75),
+        niepustynia_przy_CP['GVEG'].quantile(0.25),
+        GBD['GVEG'].quantile(0.75),
+        niepustynia_przy_GBD['GVEG'].quantile(0.25),
+    ]).mean(skipna=True), 3
+)
+
+
+print("GVEG_graniczne:", GVEG_graniczne)
+
+
+"""
+##### Rainf (wskaźnik opadów deszczu)
+"""
+
+fig, axes = plt.subplots(3, 3, figsize=(15, 15))
+
+def plot_histogram(ax, data, title, color):
+    ax.hist(data['Rainf'], bins=20, color=color, edgecolor="black")
+    ax.set_title(title)
+    ax.set_xlabel("Rainf")
+    ax.set_ylabel("Frequency")
+    ax.set_xlim(0, 220)
+    ax.set_ylim(0, 175)
```

Comment:
duplikat sygnatury poprzedniej funkcji.
To niedbałość - pewnie do naprawienia jeśli zrobilibyśmy jedną funkcję która ogarnie oba przypadki.

Reply:
Poprawione na jedną, uniwersalną funkcję.